github/github_pr_qilish.md

```diff
@@ -0,0 +1,130 @@
+```
+Kiritilgan faylimizni tekshirib olishimiz uchun `git status` dan foydalanamiz.
+```Consule
+PS D:\ochiqai\python\django\01-blog-websayt\blog-proyekt> git status
+On branch Pr
+Changes to be committed:
+  (use "git restore --staged <file>..." to unstage)
+        new file:   my_work.md
+
+Changes not staged for commit:
+  (use "git add <file>..." to update what will be committed)
+  (use "git restore <file>..." to discard changes in working directory)
+        modified:   ../../../github/github_pr_qilish.md
+```
+Faylimizni saqlab qolish uchun esa, `git commit -m`dan foydalanamiz. 
+
+```Consule
+PS D:\ochiqai\python\django\01-blog-websayt\blog-proyekt> git commit -m"Pr qo'shildi"
+[Pr 292ff81] Pr qo'shildi
+ 1 file changed, 1 insertion(+)
+ create mode 100644 django/01-blog-websayt/blog-proyekt/my_work.md
+```
+Faylni githubga birlashtirish uchun esa, `git push`ni kiritamiz.
+
+```Consule
+PS D:\ochiqai\python\django\01-blog-websayt\blog-proyekt> git push
+fatal: The current branch Pr has no upstream branch.
+To push the current branch and set the remote as upstream, use
+
+    git push --set-upstream origin Pr
+
+```
+
+Bu yerda `git push --set-upstream origin Pr` buyrug'i paydo bo'ladi. Buning ma'nosi githubga yangi branch 
+kiritayotgamizda github buni hali tanimaydi, uni ta'nitish uchun o'sha buyruqni qata kiritamiz.
+
+```Consule
+PS D:\ochiqai\python\django\01-blog-websayt\blog-proyekt> git push --set-upstream origin Pr
+Enumerating objects: 10, done.
+Counting objects: 100% (10/10), done.
+Delta compression using up to 4 threads
+Compressing objects: 100% (4/4), done.
+Writing objects: 100% (6/6), 464 bytes | 12.00 KiB/s, done.
+Total 6 (delta 3), reused 0 (delta 0), pack-reused 0
+remote: Resolving deltas: 100% (3/3), completed with 3 local objects.
+remote:
+remote: Create a pull request for 'Pr' on GitHub by visiting:
+remote:      https://github.com/ochiqai/python/pull/new/Pr
+remote:
+To https://github.com/ochiqai/python.git
+ * [new branch]      Pr -> Pr
+branch 'Pr' set up to track 'origin/Pr'.
+```
+Bunda bizga yangi branch qo'shilganligi haqida ma'lumot berilmoqda. Keyingi qadamda biz, yuborilgan xabardagi 
+birinchi manzilga `remote:      https://github.com/ochiqai/python/pull/new/Pr` boramiz.
+
+Borgan manzilimizda bu ko'rinishdagi oyna hosil bo'ladi.
+
+<p>
+    <img src=".\images\Pr.png">
+</p>
+
+Yuqorida oynada biz branch bog'lash uchun, komment qoldirishimz va reviewers taggida qaysi userlar bu fayl ko'rish ekanligi belgilaymiz.
+
+<p>
+    <img src="D:\ochiqai\python\github\images\Pr_1.png">
+</p>
+
+Keyingi qadamda oynadagi  yashil rangdagi `Create pull request` bosamiz.
+
+Keyinchalik biz bu branchni github platformasidagi `pull request`ga borib ko'rishimiz mumkin bo'ladi.
+
+<p>
+    <img src="D:\ochiqai\python\github\images\Pr_2.png"> 
+</p>
+
+Shundan so'ng biz, fayl haqida o'z fikrlarimizni qoldirish uchun `Commits` borib, matn boshidagi `+` bosamiz, va u yerda 
```

Comment:
To'g'rirog'i `Files changed` ga boramiz. Pastdagi rasmni ham o'shanga o'zgartirishimiz kerak.

Reply:
Files changedni kiritdim, rasmini o'zartira olmadim, chunki PR yopilgan edi

Reply:
Ok. Oddiy qilib quyidagi rasmni tashlasak ham bo'ladi
![image](https://user-images.githubusercontent.com/89749192/181115337-652da4bd-6560-4edf-a6b1-2f83294f4b25.png)